src/Avalonia.Diagnostics/Diagnostics/ViewModels/PropertyViewModel.cs

```diff
@@ -11,6 +11,7 @@ internal abstract class PropertyViewModel : ViewModelBase
         public abstract string Name { get; }
         public abstract string Group { get; }
         public abstract Type AssignedType { get; }
+        public abstract string IsPinned { get; set; }
```

Comment:
So... three type? Bool in ConvertBack, String here and also the Enum?... should be just one as mentioned before. Am I missing something?

Reply:
So, when it was only a bool passing it in as a groupdescription would have the groups titled true and false rather than pinned and unpinned. 
The enum was just an attempt to avoid hardcoding the strings. 

If there is another way to have the group descriptions renamed I will implement that but, I wasn't able to find anything. Even just hiding the name of the group would be a good option if possible.